developers/weaviate/concepts/indexing/inverted-indexes.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Inverted indexes
+sidebar_position: 100
+image: og/docs/indexing.jpg
+# tags: ['basics']
+---
+
+[Inverted indexes](https://en.wikipedia.org/wiki/Inverted_index) map the contents of documents to improve search performance. The Weaviate database uses inverted indexes to search object properties efficiently.
+
+There are three inverted index types in Weaviate:
+
+- [`indexSearchable`](#indexsearchable) A default index. BM25 (keyword) and hybrid search use `indexSearchable`.
+- [`indexFilterable`](#indexfilterable) A default index. Filters use the `indexFilterable` index.
+- [`indexRangeFilters`](#indexrangefilters) An optional index. Numerical filters use the `indexRangeFilter`.
+
+## Configuration
+
+These indexes are set on the property level. `indexSearchable` and `indexFilterable` indexes are enabled by default. The `indexRangeFilters` index is off by default.
+
+If you do not plan to search or filter on an object property, disable the index. Unused indexes waste space and they slow import processing.
+
+To configure an inverted index see these examples:
+
+- [`indexSearchable`](/developers/weaviate/configuration/inverted-indexes#indexsearchable)
+- [`indexFilterable`](/developers/weaviate/configuration/inverted-indexes#indexfilterable)
+- [`indexRangeFilters](/developers/weaviate/configuration/inverted-indexes#indexrangefilters)
+
+Keyword searches and hybrid searches use the BM25 ranking algorithm in conjunction with the `indexSearchable` index. To configure the BM25 algorithm, see these examples:
+
+- [BM25](/developers/weaviate/configuration/inverted-indexes#bm25)
+
+## indexSearchable
+
+[Keyword searches](/developers/weaviate/search/bm25) and [hybrid searches](/developers/weaviate/search/hybrid) use the BM25 ranking algorithm in conjunction with the `indexSearchable` index. These searches match the search terms in a query to the same term in the database. They only match terms, not meanings. To match meanings, a semantic search, use a [vector similarity search](/developers/weaviate/search/similarity).
+
+`indexSearchable` is enabled by default. This index is required for BM25 search.
+
+For simple (non-BM25) filters, the `indexFilterable` index enables faster filtering. However, if the `indexFilterable` index is not enabled, Weaviate uses the `indexSearchable` index to [filter](/developers/weaviate/concepts/prefiltering).
+
+For configuration examples, see [`indexSearchable`](/developers/weaviate/configuration/inverted-indexes#indexsearchable).
+
+## indexFilterable
+
+`indexFilterable` is enabled by default. This index is not required for filtering of BM25 search. However, this index is much faster for filtering than the `indexSearchable` index.
```

Comment:
> This index is not required for filtering of BM25 search.
I would remove this. Filtering and BM25 are independent, but linked, processes. 
Pls see https://weaviate.io/developers/weaviate/concepts/search

If someone wanted to perform "filtering of BM25 search", that sounds to me like combining a filter and a BM25 search. In which case, you would typically want `indexFilterable` to speed up the filtering. 

Just like BM25 searches can be done with `indexSearchable`, filtering can be performed without `indexFilterable`. But it would be slower.